crypto/provider_core.c

```diff
@@ -643,6 +643,21 @@ static int core_get_params(const OSSL_PROVIDER *prov, const OSSL_PARAM params[])
 static const OSSL_DISPATCH core_dispatch_[] = {
     { OSSL_FUNC_CORE_GET_PARAM_TYPES, (void (*)(void))core_get_param_types },
     { OSSL_FUNC_CORE_GET_PARAMS, (void (*)(void))core_get_params },
+
```

Comment:
NIT: maybe remove blank line

Reply:
kept the blank line, added a comment why.